secretsmanager/src/aes.h

```diff
@@ -0,0 +1,42 @@
+// Concord
```

Comment:
consider moving to utils

Reply:
This is just a convenience wrapper around Crypto++ AES code. No benefit to reuse it.